libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -957,15 +965,15 @@ void NavEKF3::UpdateFilter(void)
     }
 
     if (primary != 0 && core[0].healthy() && !armed) {
```

Comment:
should "core[0].healthy()" be changed to "core[_primary_core].healthy()"?  maybe to do this consistently we need to move the "uint8_t(_primary_core) < num_cores? _primary_core : 0;" higher.

Reply:
yep! fixed